### DISCUSSION TITLE
Support proxy config for WASM

### DISCUSSION THREAD

**Discussion by Strypper** [unoplatform/uno#12284] [Q&A]
I recently create Uno app with the `Server` include, I could start the server and the web app with it. The problem is when I started to consume the WebApi from the server in this case I use `IWeatherCache` from the template project, the doesn't take the request and it seem that it returned 304 in some scenarios. Please support this config so that the hosted web app could consume the same API hosted

**Comment by jeromelaban** [comment]
Thanks for the report. Just to be clear about what you're talking about, which proxy are you talking about?

Also, if you have a sample or set of APIs that you see missing, can you give some additional details?
